webpack.config.ts

```diff
@@ -29,6 +29,7 @@ module.exports = [
             filename: "[name].bundle.js",
             path: path.resolve(__dirname, "out/host"),
         },
+        target: "node",

```

Comment:
You can't add this here. The host bundle that this is building is designed to run in the webview, which means it is definitely not running under node! I don't know what you were trying to fix here, but there needs to be another way to do it, since the host bundle needs to be compiled as a browser target not node.

Reply:
This is the line where I spent most of my time 🍶 , this was to workaround an issue that 'fs' module (used by vscode-nls) has with webpack. As I'm removing vscode-nls this is not needed anymore.